runtime/lua/nvim/health.lua

```diff
@@ -50,7 +50,7 @@ local function check_config()
 
   local init_lua = vim.fn.stdpath('config') .. '/init.lua'
   local init_vim = vim.fn.stdpath('config') .. '/init.vim'
-  local vimrc = vim.env.MYVIMRC or init_lua
+  local vimrc = vim.fn.expand(vim.env.MYVIMRC or init_lua)
```

Comment:
`init_lua` should not be expanded

Reply:
I did it, because `expand` on a non-existing variable returns `v:null` and so `or` isn't triggered.

Is having pseudo-ternary like this is fine?

```suggestion
  local vimrc = vim.env.MYVIMRC and vim.fn.expand(vim.env.MYVIMRC) or init_lua
```

Reply:
Yes